interpreter/core/respond.py

```diff
@@ -113,9 +114,21 @@ def respond(interpreter):
 
                 # Get a code interpreter to run it
                 language = interpreter.messages[-1]["language"]
-                if language not in interpreter._code_interpreters:
-                    interpreter._code_interpreters[language] = create_code_interpreter(language)
-                code_interpreter = interpreter._code_interpreters[language]
+                if language in language_map:
+                    if language not in interpreter._code_interpreters:
+                        interpreter._code_interpreters[language] = create_code_interpreter(language)
+                    code_interpreter = interpreter._code_interpreters[language]
+                else:
+                    #This still prints the code but don't allow code to run. Let's Open-Interpreter know through output message
+                    print(f"Error: Open-Interpreter do not currently support {language}.")
+
+                    interpreter.messages[-1]["output"] = ""
+                    output = "\n" + f"Error: Open-Interpreter do not currently support {language}."
```

Comment:
Can we change these to the following message:

```python
error_output = f"Error: Open Interpreter does not currently support {language}."
```

And then use `error_output` in both places instead of repeating the string?